### PR TITLE
Document experimental support for other architectures

### PR DESCRIPTION
**Pull request by luxas**
Document our new `hyperkube` images. `arm64` and `ppc64le` will be available when `v1.3.0-alpha.3` is released.

@vishh @david-mcmahon @brendandburns @johndmulhausen

### CODE REVIEW

File: docs/getting-started-guides/docker.md [hunk 1]
```diff
@@ -56,6 +54,8 @@ docker run \
 
 > If you would like to mount an external device as a volume, add `--volume=/dev:/dev` to the command above. It may however, cause some problems described in [#18230](https://github.com/kubernetes/kubernetes/issues/18230)
 
+> Other architectures than `amd64` are experimental and sometimes unstable, but feel free to try them out! Valid values: `arm`, `arm64` and `ppc64le`. ARM is available with Kubernetes version `v1.3.0-alpha.2` and higher. ARM 64-bit and PowerPC 64 little-endian are available with `v1.3.0-alpha.3` and higher. Track as progress is being made in [this proposal](https://github.com/kubernetes/kubernetes/issues/17981)
```
Comment (vishh) [inline docs/getting-started-guides/docker.md]: nit: Rephrase as `Architectures other than `amd64` are experimental`

Comment (vishh) [inline docs/getting-started-guides/docker.md]: @luxas You are too quick :) Check out the second part of my comment.

**Comment by vishh** [comment]
Technical contents LGTM. cc @johndmulhausen

File: docs/getting-started-guides/docker.md (hunk 1 above)
Comment (luxas) [inline docs/getting-started-guides/docker.md]: Hahaha :smile:
Thanks for the comments, it's late at night here already so it's good to have someone thinking about how to say things the best way :)

**Comment by luxas** [comment]
@vishh Clarified the `kubectl` download section even more based on user feedback. 
PTAL when you have time

Also ping @johndmulhausen, we should get this in ASAP

**Comment by johndmulhausen** [comment]
@luxas looks like you have a merge conflict that needs resolving. :(

**Comment by luxas** [comment]
@johndmulhausen Rebased. Please merge :)

**Comment by luxas** [comment]
ping @johndmulhausen @vishh 
let's get this in

**Comment by luxas** [comment]
@bgrant0607 Mind take a look?
This has been pending for far too long although it's LGTM
I know I can merge it myself, but since one shouldn't self-merge things, I'm leaving it for some other maintainer

**Comment by bgrant0607** [comment]
Thanks, @luxas. LGTM

**Pull request merged**